tests/kernel/boot_ltp.pm

```diff
@@ -131,6 +132,14 @@ EOF
             'systemctl --no-pager -p Id show network.service | grep -q Id=wicked.service && { export ENABLE_WICKED=1; systemctl disable wicked; }'
         );
 
+        # Also disable firewall
+        if (is_sle('<15')) {
+            script_run('systemctl disable SuSEfirewall2; systemctl stop SuSEfirewall2');
+        }
+        elsif (is_sle('15+')) {
+            script_run('systemctl disable firewalld; systemctl stop firewalld');
+        }
+
```

Comment:
opensusebasetest contains `firewall` function which returns firewall implementation based on the product, so you don't need to do own version check

Reply:
Fixed.